docs/02_getting-started/02_development-environment/02_introduction.md

```diff
@@ -1,37 +1,29 @@
 ---
-content_title: "1.1: Introduction"
-link_text: "1.1: Introduction"
+content_title: "1.1: Prerequisites"
```

Comment:
This is not an introduction for eosio but also not only contains prerequisite. It is more of an introduction to this tutorial.